recipes/jungle/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.11)
+set(CMAKE_CXX_STANDARD 11)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(jungle REQUIRED)
```

Comment:
```suggestion
find_package(jungle CONFIG REQUIRED)
```

Reply:
This is the only review change I'm not quite sure I grok. Isn't this only needed when `cmake_find_package_multi` is used?